tree/ntuple/v7/doc/specifications.md

```diff
@@ -257,6 +257,8 @@ The type can take one of the following values
 
 Each locator type follows a given format for the payload (see Section "Well-known payload formats" below).
 The range 0x03 - 0x7f is currently unused.
+The DAOS locator and the Object64 payload format are currently reserved but unspecified.
```

Comment:
Do we actually want to spell out that we (might) want to implement a DAOS locator? From a specification standpoint we are technically only reserving locator flag 0x02, but if we remove the Object64 payload format from the specs it seems to me that we should also remove its mention here and just say that flag 0x02 is reserved for future use

Reply:
True... we don't even need to specifically reserve it then. We just reserve `0x03 - 0x7f` for future use.